NEWS.md

```diff
@@ -1,178 +1,246 @@
 # lintr (development version)
 
-## Potentially breaking changes and pending deprecations
+## Breaking changes
+
+* Exclusions specified in the `.lintr` file are now relative to the location of that file 
+  and support excluding entire directories (#158, #438, @AshesITR)
+* Consistent access to linters through a function call, even for linters without parameters
```

Comment:
Slightly confused about what goes in breaking vs. deprecated

earlier when I read, I thought what should go here are backwards-incompatible changes, because we mention some function deprecations (like `absolute_paths_linter`) but not others (like `semicolon_terminator_linter`), and the difference is deprecations here are not back-compatible.

but this change (consistent access...) is backwards-compatible, in that we warn for providing linters as symbols, for now.

maybe the sections should be combined, and one bullet point is linter deprecations, with the removals coming first?